src/AccessLogEntry.h

```diff
@@ -44,11 +44,11 @@ class AccessLogEntry: public CodeContext
     typedef RefCount<AccessLogEntry> Pointer;
 
     AccessLogEntry();
-    virtual ~AccessLogEntry();
+    ~AccessLogEntry() override;
```

Comment:
In early C++17 days, there were [debates](https://github.com/isocpp/CppCoreGuidelines/issues/721) regarding marking destructors with `override`/`final`. That debate has been [settled](https://github.com/isocpp/CppCoreGuidelines/pull/1448) in favor of marking. Clang-tidy and this change follow that settled recommendation ([C.128](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c128-virtual-functions-should-specify-exactly-one-of-virtual-override-or-final)).